client/blocks/login/login-form.jsx

```diff
@@ -448,6 +448,36 @@ export class LoginForm extends Component {
 		);
 	}
 
+	renderMagicLoginLink() {
+		if (
+			! canDoMagicLogin(
+				this.props.twoFactorAuthType,
+				this.props.oauth2Client,
+				this.props.wccomFrom,
+				this.props.isJetpackWooCommerceFlow
+			)
+		) {
+			return null;
+		}
+
+		return this.props.translate(
+			' Would you like to {{magicLoginLink}}receive a login link{{/magicLoginLink}}?',
```

Comment:
```suggestion
			'Would you like to {{magicLoginLink}}receive a login link{{/magicLoginLink}}?',
```

It's also not clear where the user will receive the link. I would add "to your email".

Reply:
> It's also not clear where the user will receive the link. I would add "to your email".

The user is redirected to `/wp-login.php?action=lostpassword` where the user gets more instructions about the next steps (see screenshot). So adding "to your email" might be a bit overkill? 

@ollierozdarz, WDYT?


<img width="547" alt="Screenshot 2021-07-13 at 09 40 09" src="https://user-images.githubusercontent.com/12104589/125411564-538d6980-e3be-11eb-8063-7b4b84895185.png">

Reply:
I removed the leading space in ecf6a0c44a39f1e802ddadfc09ec43fb269ee84f

Reply:
I see. But I'd add the space outside of the translation. It's all optional though.